modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc

```diff
@@ -0,0 +1,201 @@
+
+The performance information provided in this topic is based on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. The cluster was configured with three m4.large controller nodes and three r4.large worker nodes. M4 instances feature a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2. The processors run at a base clock rate of 2.4 GHz, and can go as high as 3.0 GHz with Intel Turbo Boost. The load generator used was hosted on a separate AWS instance in the same region.
+
+You can allocate the following resources when deploying an application:
+
+* vCPU cores
+** `Reserved vCPU`
++
+The amount of vCPU guaranteed to the application and reserved for its use.
+** `vCPU Limit`
++
+The maximum amount of vCPU the application can use (the level to which it can burst). This is shared CPU on the worker node.
+
+* Memory
+// end::description[]
+// tag::cpuLicensedCores[]
+== CPU and Licensed Cores
+
+The physical CPU core capacity often differs from the MuleSoft license core amount.
+
+* The physical CPU core capacity is the CPU capacity provided by Runtime Fabric worker nodes, which is displayed as vCPU utilization in the Runtime Fabric console.
+* The MuleSoft license core amount is the entitlement needed for Mule application deployment.
+
+You cannot purchase fractional MuleSoft license core amounts. For example, if you plan to have 1.5 cores available, you must purchase a MuleSoft license for 2 cores.
+
+The underlying Runtime Fabric cluster does not need to be licensed. The actual application core utilization must be less than or equal to the MuleSoft license core amount you purchased. To measure utilization, use the dashboard in Ops Center.
+
+=== MuleSoft License Compliance
+
+You are responsible for ensuring that you remain compliant with your MuleSoft licensing. To remain compliant, ensure that you do not exceed the number of CPU cores provided by your MuleSoft license at any given time.
+
+When using Runtime Fabric on VMs/Bare Metal, use the Core CPU Usage dashboard in Ops Center. This dashboard shows core CPU usage for Mule applications only, so it can help you determine your core CPU usage across all your Mule applications.
+
+To check core CPU usage:
+
+. From the Ops Center navigation, click *Monitoring*.
+. From the dashboard dropdown list, select *CPU Core Usage*. 
+. From the time picker, select your date range. 
++
+You might see multiple data points for a certain time frame depending on data granularity. To get an accurate assessment of core CPU usage against licensing, look for the highest number within the selected time frame. 
+// end::cpuLicensedCores[]
+// tag::vcpuAllocation[]
```

Comment:
Ops center monitoring is only available on gravitational appliances. For BYOK customers typically implement/leverage CPU/memory monitoring solutions from the respective cloud providers.